repository/src/main/java/com/michaldrabik/repository/images/EpisodeImagesProvider.kt

```diff
@@ -23,40 +24,27 @@ class EpisodeImagesProvider @Inject constructor(
   private val dispatchers: CoroutineDispatchers,
   private val remoteSource: RemoteDataSource,
   private val localSource: LocalDataSource,
-  private val mappers: Mappers
+  private val mappers: Mappers,
+  private var translationsRepository: TranslationsRepository
 ) {
 
   suspend fun loadRemoteImage(showId: IdTmdb, episode: Episode): Image = withContext(dispatchers.IO) {
-    val tvdbId = episode.ids.tvdb
     val tmdbId = episode.ids.tmdb
```

Comment:
This is used only once actually so can be inlined.

Reply:
Perfect! I changed it without problems, the truth is that I only kept it that way to follow the ‘style’ of the rest of the providers.